automation/run-tests.sh

```diff
@@ -274,6 +259,14 @@ done
 : ${TEST_TYPE:=$TEST_TYPE_ALL}
 : ${CONTAINER_IMAGE:=$FEDORA_IMAGE_DEV}
 
+TOX_ARGUMENT=""
+
+if [[ $CONTAINER_IMAGE == *"centos"* ]]; then
+    # In CentOS, instead of using pip installed nmstate dependencies,
+    # we would like to use OS site packages.
+    TOX_ARGUMENT="--sitepackages"
+fi
+
```

Comment:
Is this still needed now?

Reply:
Replaced it with a statement in tox, specific for the pylint target (all others do not need it).